sympy/core/sympify.py

```diff
@@ -66,6 +66,9 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
        - booleans, including ``None`` (will leave ``None`` unchanged)
        - lists, sets or tuples containing any of the above
 
+    Note that this function uses ``eval``, and thus shouldn't be used on
```

Comment:
@jtoka or @asmeurer, feel free to fix in another pr. i merged before this comment came in.